AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -45,6 +45,7 @@ vc.ru##[air-module="module.teaserTest"]
 ! https://uploads.adguard.com/up04_vv5cc_vc.ru__biznes_tehnologii_idei_modeli_rosta_startapy.png
 vc.ru##.teaserTinkoffStock
 ! https://github.com/AdguardTeam/AdguardFilters/issues/31397
+hi-fi.ru##a[href^="https://zen.yandex.ru"]
```

Comment:
plz move to `!* Subscriptions/newsletters (not popups)` section in this file, you can add new domain to existing rule